src/data/history.ts

```diff
@@ -153,6 +162,9 @@ const processLineChartEntities = (
 
       if (
         processedStates.length > 1 &&
+        processedState.attributes &&
+        processedStates[processedStates.length - 1].attributes &&
+        processedStates[processedStates.length - 2].attributes &&
```

Comment:
If we don't have attributes on all we should still check if the states are the same to not push the same point to the chart?

Reply:
So I think we should change the `equalState` function instead of only calling that when we have attributes

Reply:
Done, and reverted this change